src/data/finset/basic.lean

```diff
@@ -1823,6 +1823,19 @@ end
   to_finset l.reverse = l.to_finset :=
 to_finset_eq_of_perm _ _ (reverse_perm l)
 
+lemma repeat_to_finset_zero {a : α} : (repeat a 0).to_finset = (∅:finset α)
+  := by rw [list.repeat.equations._eqn_1 a, list.to_finset_nil]
+
+lemma repeat_to_finset_pos {a : α} {n : ℕ} (hn : 0 < n):
+  (list.repeat a n).to_finset = {a}
+  :=
```

Comment:
```suggestion
lemma to_finset_repeat_of_pos (a : α) {n : ℕ} (hn : 0 < n) :
  (list.repeat a n).to_finset = {a} :=
```
This is about `to_finset (list.repeat a n)`, so `to_finset` should appear first in the name like it does above.